src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -276,8 +276,8 @@ export function PriceChart({ width, height, prices }: PriceChartProps) {
   const crosshairEdgeMax = width * 0.85
   const crosshairAtEdge = !!crosshair && crosshair > crosshairEdgeMax
 
-  /* Default curve doesn't look good for the HOUR/ALL chart */
-  const curveTension = timePeriod === TimePeriod.ALL ? 0.75 : timePeriod === TimePeriod.HOUR ? 1 : 0.9
+  /* Default curve doesn't look good for the HOUR chart */
```

Comment:
Can you make this comment more informative? Do you know _why_ it doesn't look good?
Including that could help future maintainers who might add more TimePeriod choices.

Reply:
sounds good done